java/src/com/google/idea/blaze/java/run/BlazeJavaRunProfileState.java

```diff
@@ -280,9 +285,12 @@ static BlazeCommand.Builder getBlazeCommandBuilder(
   private ProcessHandler getScopedProcessHandler(
       Project project, ImmutableList<String> command, WorkspaceRoot workspaceRoot)
       throws ExecutionException {
+    GeneralCommandLine commandLine = new GeneralCommandLine(command);
+    Map<String, String> envVars = getState(getConfiguration()).getUserEnvVarsState().getData().getEnvs();
```

Comment:
same as before: I'm not sure if this is needed but in https://github.com/ujohnny/intellij/blob/master/clwb/src/com/google/idea/blaze/clwb/run/BlazeCidrLauncher.java#L239 the ParentEnvironmentType is set. do we need to do something similar here?